merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -244,14 +252,26 @@ public SimulationResults runSimulation(final CreateSimulationMessage message)
           "No mission model configuration defined for mission model. Simulations will receive an empty set of configuration arguments.");
     }
 
-    // TODO: [AERIE-1516] Teardown the mission model after use to release any system resources (e.g. threads).
-    return SimulationDriver.simulate(
-        loadAndInstantiateMissionModel(message.missionModelId(), message.simulationStartTime(), SerializedValue.of(config)),
-        message.activityDirectives(),
-        message.simulationStartTime(),
-        message.simulationDuration(),
-        message.planStartTime(),
-        message.planDuration());
+    try (final var simulationExtentListener =
+        ThreadedSimulationExtentReporter
+            .spawn(
+                "ThreadedSimulationExtentReporter",
+                writer::reportSimulationExtent,
+                simulationExtentMinimumPollPeriodMillis
```

Comment:
Why is this specifically called the _minimum_ poll period instead of just the poll period?

Reply:
I think we can remove it without adding too much confusion :+1: It is, strictly speaking, a minimum, because it is possible that the time it takes to write to the database could exceed the poll period. In that scenario, it would issue the next request as soon as the previous completes.